continue-reading.html

```diff
@@ -1,13 +1,231 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Motherflippin Homepage</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+
+<body id="container">
+
+
+
+
+  <!-- header section -->
+  <div class="item header">
+    <a href="index.html">
+      <h1>
+        C O N T I N U E &nbsp R E A D I N G
+      </h1>
+    </a>
+    <div class="subhead">
+      GO ON -- KEEP READING
+
+      <div class=headrln>
+        <br> <br />
+        <hr>
+        <hr />
+      </div>
+    </div>
+
+  </div>
+
+
+  <!--please ignore this, definitely have to do some tweaking here. SORRY!-->
+
+
+  <!-- <section class="item features">
+    <div class="featsect">
+      <h4 style="text-align: center;"> Much art. Such beauty. Here's an inspirational quote for ya. Feel inspired? Damn
+        right you
+        do.</h4>
+      <div class="contimg">
+        
+        <img class="img" src=images/abstract-q-g-640-480-6.jpg title="generic shit">
+
+      </div>
+
+      <a href="learn-more.html" class='button'>
+        <button type="button"> Tell me more! </button>
+      </a>
+
+      <h3 class='banner'>banner quote: im inspirational as fuq, just look at these pictures </h3>
+      
+    </div>
+
+  </section> -->
+
+  <section class="item features"> Please ignore the awkward amount of space - this will be better... Yikes this is a
+    lot. Damn you, grid!!! Well, at least I'm learning.
+
+

```

Comment:
This has so much white space because of the size of your grid, adjusting the size of the grid and the size of the whole page will adjust the size of the whitespace here.